src/pickit.py

```diff
@@ -20,7 +20,7 @@ def __init__(self, screen: Screen, item_finder: ItemFinder, ui_manager: UiManage
         self._belt_manager = belt_manager
         self._ui_manager = ui_manager
         self._game_stats = game_stats
-        self._config = Config()
+        self._config = self._item_finder._config
```

Comment:
Underscore means that it is supposed to be a private variable. Either remove the underscore of config in item finder or write a getter for it.

Reply:
Yeah, I just added a separate config to pass in for this